ui/tests/e2e/search_log.test.ts

```diff
@@ -0,0 +1,52 @@
+import puppeteer from 'puppeteer'
+import ppExpect from 'expect-puppeteer'
+import { LOGIN_URL, PUPPETEER_CONFIG } from './test_config'
+
+describe('Search Logs', () => {
+  let browser
+  beforeAll(async () => {
+    browser = await puppeteer.launch(PUPPETEER_CONFIG)
+  })
+
+  afterAll(() => {
+    browser.close()
+  })
+
+  it(
+    'should search correct logs',
+    async () => {
+      const page = await browser.newPage()
+
+      // login
+      await page.goto(LOGIN_URL)
+      await ppExpect(page).toClick('button#signin_btn')
+
+      // jump to search logs page
+      await ppExpect(page).toClick('a[href="#/search_logs"]')
```

Comment:
prefer not to match by text or id

Reply:
I guess you mean prefer to match by text or id, right? fixed by that.